docs/distributed/REBALANCE.md

```diff
@@ -0,0 +1,74 @@
+# Pool rebalance
+
+## Introduction
+A MinIO distributed setup consists of one or more server pools. MinIO distributes objects uploaded among these pools proportional to their available drive space. A good proportion of objects uploaded will land on the recently added pools. Over time, objects would be uniformly distributed across all the pools.
+
+Certain workloads may require that objects are distributed uniformly across all pools as much as possible, at all times. To support this, MinIO needs to redistribute objects from existing pools to the newly added pools. This is now possible with the Pool rebalance feature.
+
```

Comment:
```suggestion

This is generally a time and IO consuming operation, so it can potentially be disruptive for your cluster. We recommend [consulting with a MinIO engineer](https://min.io/pricing) before deciding on starting this operation.

```